README.md

```diff
@@ -240,23 +399,17 @@ class VeryCustomNibBasedCell: UITableViewCell, NibReusable {
 }
 ```
 
-## Advantages of using Reusable
-
-Note how, in the examples above and when using `Reusable`:
+The same is true for all the protocols of this pod, which always provide default implementations which could still be replaced by your own if you need some custom cases.
 
-* You never have to use a String-typed `reuseIdentifier` in your code anywhere
-* The **proper cell type to dequeue is infered by Swift from the return type**
+_But the beauty is in 90% of cases the default implementation will match typical conventions and the default implementations will be exactly what you want!_
 
-  * The simple fact that we wrote `dequeueReusableCell(…) as MyCellType` let the Swift compiler infer that you expect a `MyCellType` and deduce the `reuseIdentifier` to use for that all by itself! ✨
 
-* The code is type-safe, as the `dequeueReusableCell(…)` function will return the type you asked — and not the `UITableViewCell` non-specific superclass as with Apple APIs
 
-  * This way you can call methods even specific to your `MyCellType` return type on that cell next, without the need to cast it!
+---
 
-For more information and explanations on this code, see [my detailed blog post about this here](http://alisoftware.github.io/swift/generics/2016/01/06/generic-tableviewcells/).
 
 
-## Example Project
+# Example Project
 
 This repository comes with an example project in the `Example/` folder. Feel free to try it.
 
```

Comment:
Can't comment on it directly, but the line below this says "It demonstrate", which should be "It demonstrates"